src/subcommand/init.rs

```diff
@@ -215,6 +226,38 @@ _z_cd() {{
 use posix_z as bash_z;
 use posix_z as zsh_z;
 
+fn powershell_z(z_cmd: &str) -> String {
+    format!(
+        r#"
+function {} {{
+    function z_cd($dir) {{
+        try {{
+            Set-Location $dir -ea Stop
+            if ($env:_ZO_ECHO -eq "1") {{
```

Comment:
Currently, the other shells simply check if `_ZO_ECHO` is set to a non-empty string, rather than a specific value like `1`. It would be great if PowerShell also matches this behaviour.

Reply:
I can change this, but I'm not a fan of the value `0` being treated equivalently. I do think it's common to test the value `1` though.

Reply:
That's a fair point. I will change the behaviour of the other shells to match this, then.

Reply:
Other shells now require `_ZO_ECHO` to be set to `1` too, so I'm marking this as resolved.